app/views/questionnaire.py

```diff
@@ -281,6 +283,20 @@ def _save_sign_out(collection_id, eq_id, form_type, q_manager, this_location):
         return redirect(url_for('.get_sign_out', eq_id=eq_id, form_type=form_type, collection_id=collection_id))
 
 
+def _compare_and_remove_repeating_on_household_if_answers_changed(answer_store, group_id):
+
+    household_answer_store = answer_store.filter(block_id="household-composition")
```

Comment:
household_answer_store suggests it's something other than an array, just household_answers would be more meaningful